README.md

```diff
@@ -20,17 +20,16 @@ See https://gradben.ch for a daily overview of all the tools (columns) and bench
 
 ## Usage
 
-If you have [Git][] installed, you can clone this repository; for instance, via the [GitHub CLI][]:
+If you have [Rust][] installed, you can download and install the GradBench CLI:
 
 ```sh
-gh repo clone gradbench/gradbench
-cd gradbench
+cargo install --locked --git https://github.com/gradbench/gradbench
```

Comment:
It is odd to install the `gradbench` CLI like this, since anyone actually using GradBench will presumably have a checkout of the repository. Apart from putting binaries outside of the GradBench git checkout itself, this also runs the risk of people using an outdated `gradbench` CLI. There is a very low chance of actually getting the `gradbench` CLI into any package repositories, so the documentation and the architecture should be written such that this is not the expected outcome.

The instructions in CONTRIBUTING.md are a better workflow.